descwl_shear_sims/shifts.py

```diff
@@ -230,6 +253,46 @@ def get_random_shifts(*, rng, dim, buff, size):
     return shifts
 
 
+def get_random_circle_shifts(*, rng, dim, buff, size):
+    """Gets a set of gridded shifts in a circle, with random shifts at the
```

Comment:
not gridded right?  Maybe "get a set of random shifts on a disk"?

Reply:
Done